save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt

```diff
@@ -172,6 +171,12 @@ class VulnerabilityMetadataService(
                     getScore(it)
                 } ?: 0f
 
-        private fun Boolean.toVulnerabilityStatus() = if (this) VulnerabilityStatus.AUTO_APPROVED else VulnerabilityStatus.PENDING_REVIEW
+        private fun Boolean.toVulnerabilityStatus(isNew: Boolean) = if (this) {
+            VulnerabilityStatus.AUTO_APPROVED
+        } else if (isNew) {
+            VulnerabilityStatus.CREATED
+        } else {
+            VulnerabilityStatus.PENDING_REVIEW
+        }
```

Comment:
Maybe change this function to something like `private fun getVulnerabilityStatus(isAutoApprove: Boolean, isNew: Boolean)`?

Reply:
Ok, changed it